------------------------------------------------------------
revno: 14166
revision-id: squid3@treenet.co.nz-20170529125748-qt7yhdloygl4xosg
parent: squid3@treenet.co.nz-20170529071037-o91o8xvaqata5y2b
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Tue 2017-05-30 00:57:48 +1200
message:
  Revert r14161
  
  Wrong patch and commit message.
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170529125748-qt7yhdloygl4xosg
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: ddecde537486c58df04564f3818b8ad9929dd186
# timestamp: 2017-05-29 13:51:06 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170529071037-\
#   o91o8xvaqata5y2b
# 
# Begin patch
=== modified file 'src/tunnel.cc'
--- src/tunnel.cc	2017-05-29 05:33:59 +0000
+++ src/tunnel.cc	2017-05-29 12:57:48 +0000
@@ -836,7 +836,7 @@
  * Call the tunnelStartShoveling to start the blind pump.
  */
 static void
-tunnelConnectedWriteDone(const Comm::ConnectionPointer &conn, char *, size_t len, Comm::Flag flag, int, void *data)
+tunnelConnectedWriteDone(const Comm::ConnectionPointer &conn, char *buf, size_t size, Comm::Flag flag, int xerrno, void *data)
 {
     TunnelStateData *tunnelState = (TunnelStateData *)data;
     debugs(26, 3, HERE << conn << ", flag=" << flag);
@@ -848,11 +848,6 @@
         return;
     }
 
-    if (ClientHttpRequest *http = tunnelState->http.get()) {
-        http->out.headers_sz += len;
-        http->out.size += len;
-    }
-
     tunnelStartShoveling(tunnelState);
 }
 

